crates/composefs/src/repository.rs

```diff
@@ -383,15 +401,21 @@ impl<ObjectID: FsVerityHashValue> Repository<ObjectID> {
         self.write_image(Some(name), &data)
     }
 
-    pub fn open_image(&self, name: &str) -> Result<OwnedFd> {
+    fn open_image(&self, name: &str) -> Result<(OwnedFd, bool)> {
```

Comment:
Depending on how many users this API has, it might make sense to inline it where it's used... I guess the GC thing is using it as well, though......

I'm glad it's no longer public, at least.  A short doc about what the 'bool' is would certainly be appreciated though.